packages/framework-integration-tests/integration/cli/setup.ts

```diff
@@ -29,6 +34,7 @@ before(async () => {
 
   try {
     await Promise.all(removeFiles(testFiles))
+    await Promise.all(removeFolders(testFolders))
```

Comment:
This code might be slightly faster (probably not really significative, but... 🤷) if you concat the two arrays and call `Promise.all` only one time:

```suggestion
    await Promise.all([
      ...removeFiles(testFiles),
      ...removeFolders(testFolders),
    ])
```